Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -7523,6 +7523,17 @@
         }
       }
     },
+    "notFindingResults" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "We are not finding results in our immediate network for your search. You can continue waiting or try another search term."
```

Comment:
I'm not sure what "in our immediate network" means. We send searches to all relays on the user's relay list, so I'd suggest changing this to "We are not finding results on your relays" or just dropping the phrase to something like "This is taking a long time. You can continue waiting or try adjusting your search term."